### PR TITLE
Install Decidim Awesome module

### PR DESCRIPTION
**Pull request by laurajaime**
Module:https://github.com/Platoniq/decidim-module-decidim_awesome

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -649,6 +687,21 @@
     t.index ["decidim_user_id"], name: "index_decidim_impersonation_logs_on_decidim_user_id"
   end
 
+  create_table "decidim_ldap_configurations", force: :cascade do |t|
```
Comment (tramuntanal) [inline db/schema.rb]: This should not be here, it belongs to DiBa only

**Review by tramuntanal — CHANGES_REQUESTED** [review]
I see an LDAP related table in the `schema.rb`, this table should not be in Pinto neither due to this PR nor due to #7

File: db/schema.rb (hunk 1 above)
Comment (laurajaime) [reply db/schema.rb]: Ops, I don't know why is it here.

**Review by tramuntanal — APPROVED** [review]
LGTM!

**Pull request merged**